extra/mariabackup/xtrabackup.cc

```diff
@@ -5461,19 +5452,20 @@ static bool xtrabackup_backup_func()
 
 	/* cd to datadir */
 
-	if (my_setwd(mysql_real_data_home,MYF(MY_WME)))
-	{
+	if (my_setwd(mysql_real_data_home,MYF(MY_WME))) {
```

Comment:
Our code formatting style disallows K&R style `{` placement, except for the body of a `switch` statement.